python/requirements.txt

```diff
@@ -83,6 +81,7 @@ pytest-sugar
 pytest-lazy-fixture
 pytest-timeout
 pytest-virtualenv
+redis >= 3.5.0, < 4.0.0
```

Comment:
Do we still need this? If yes, why?

Reply:
Yes. Python Redis dependency is still used in two places:
1. Bootstrapping Ray with external Redis.
2. `test_ray_init.py`
Both can be removed in theory, but (1) is more involved.

Reply:
This is only for testing. We've already removed redis libs as ray direct dependence from setup.py